packages/pymedphys_dicom/src/pymedphys_dicom/dicom/anonymise.py

```diff
@@ -40,77 +40,276 @@
 
 
 IDENTIFYING_KEYWORDS = ("AccessionNumber",
```

Comment:
I think we should move this out into a json file, and then in the docs link to this json file to say "this is what DICOM anonymiser will remove make sure you are comfortable with these tags, and only these tags being removed/anonymised."

Reply:
Sure :)

What about `BaselineDicomDictionary` and `RepeatersDictionary`?

Reply:
I certainly wouldn't be opposed to having those moved out, but the main need I believe initially is certainly to be able to have a clear file that can be linked to from the docs for the anonymisation tags.